File: 프로그래머스/0/120807. 숫자 비교하기/숫자 비교하기.py
def solution(num1, num2):
    answer = 0  # 결과 값을 저장할 변수를 초기화

    if num1 == num2:  # 두 숫자가 같은 경우
        answer = 1  # 결과 값에 1을 저장
    else:  # 두 숫자가 다른 경우
        answer = -1  # 결과 값에 -1을 저장

    return answer  # 최종 결과 반환

# 예제 입력 및 실행 결과 출력
print(solution(2, 3))       # -1 (2와 3은 다르므로 -1 반환)
print(solution(11, 11))     # 1 (11과 11은 같으므로 1 반환)
print(solution(7, 99))      # -1 (7과 99는 다르므로 -1 반환)

File: 프로그래머스/0/120807. 숫자 비교하기/숫자 비교하기.java
class Solution {
    public int solution(int num1, int num2) {
        int answer = 0;

        if (num1 == num2) {
            answer = 1;
        } else {
            answer = -1;
        }

        return answer;
    }

    public static void main(String[] args) {
        Solution s = new Solution();

        // 테스트 예제 실행
        System.out.println(s.solution(2, 3));  // -1
        System.out.println(s.solution(11, 11)); // 1
        System.out.println(s.solution(7, 99));  // -1
    }
}